internal/security/setup/option/optionHelpers_test.go

```diff
@@ -0,0 +1,128 @@
+//
+// Copyright (c) 2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License. You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License
+// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied. See the License for the specific language governing permissions and limitations under
+// the License.
+//
+// SPDX-License-Identifier: Apache-2.0'
+//
+
+package option
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/edgexfoundry/edgex-go/internal/security/setup"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetWorkDir(t *testing.T) {
+	// sets up configuration for SecretsSetupInfo
+	tearDown := setupGenerateTest(t)
+	defer tearDown(t)
+
+	origEnvVal := os.Getenv(envXdgRuntimeDir)
+	os.Unsetenv(envXdgRuntimeDir)
+
+	// test WorkDir that's configured by toml
+	runTimeDir := getWorkDir()
+	assert.Equal(t, "./edgex/security-secrets-setup", runTimeDir)
+
+	// test default WorkDir
+	oldConfig := setup.Configuration
+	setup.Configuration = &setup.ConfigurationStruct{}
+	setup.Configuration.SecretsSetup = setup.SecretsSetupInfo{
+		WorkDir: "",
+	}
+	runTimeDir = getWorkDir()
+	assert.Equal(t, filepath.Join(defaultWorkDir, pkiInitBaseDir), runTimeDir)
+	setup.Configuration = oldConfig
+
+	// test env variable
+	os.Setenv(envXdgRuntimeDir, "/run")
+	runTimeDir = getWorkDir()
+	assert.Equal(t, filepath.Join("/run", pkiInitBaseDir), runTimeDir)
+
+	os.Setenv(envXdgRuntimeDir, origEnvVal)
+}
+
+func TestGetCertConfigDir(t *testing.T) {
+	// sets up configuration data for SecretsSetupInfo
+	tearDown := setupGenerateTest(t)
+	defer tearDown(t)
```

Comment:
Sorry, I miss this- can you also change this to remove `t` on the returned `tearDown` func?  So this makes all tearDown more consistent